prow/config.yaml

```diff
@@ -449,6 +449,7 @@ tide:
     kubernetes-sigs/cluster-api-provider-openstack: squash
     kubernetes-sigs/cluster-api-provider-vsphere: squash
     kubernetes-sigs/kubespray: squash
+    kubernetes/cloud-provider-openstack: squash
```

Comment:
i made the minor complain that this functionality nukes GPG keys and SOB sign-offs in multi-commit PRs.
repo maintainers should be wary of picky contributors, but if the repo is fine with that SGTM.

Reply:
:100:
/approve
/hold
please ensure the maintainers are OK with this before merging (`/hold cancel`) :-)